src/os/bluestore/StupidAllocator.cc

```diff
@@ -258,7 +258,7 @@ double StupidAllocator::get_fragmentation(uint64_t alloc_unit)
   uint64_t intervals = 0;
   {
     std::lock_guard<std::mutex> l(lock);
-    max_intervals = num_free / alloc_unit;
+    max_intervals = p2roundup(num_free, alloc_unit) / alloc_unit;
```

Comment:
'p2roundup' seems to be unavailable in Luminous, I used 'P2ROUNDUP' and it worked for me.

Reply:
> 'p2roundup' seems to be unavailable in Luminous, I used 'P2ROUNDUP' and it worked for me.

yep, addressed